web-js/src/components/useTodos_local.js

```diff
@@ -0,0 +1,67 @@
+import { BSON } from "realm-web";
+import React from "react";
+import { useRealmApp } from "./RealmApp";
+
+const createExampleTodos = (userId = "60810749247a41a9809fba46") => [
+  {
+    _id: new BSON.ObjectID(),
+    _partition: userId,
+    summary: "Do the dishes",
+    isComplete: false,
+  },
+  {
+    _id: new BSON.ObjectID(),
+    _partition: userId,
+    summary: "Buy groceries",
+    isComplete: false,
+  },
+];
+
+function latency(ms) {
+  return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+export function useTodos() {
+  const realmApp = useRealmApp();
+  const [todos, setTodos] = React.useState([]);
+  
+  const [loading, setLoading] = React.useState(true);
+  React.useEffect(() => {
+    const fetchTodos = async () => {
+      await latency(1640)
```

Comment:
What's up with this artificial timer?

Reply:
This is the "local" todo provider which mocks everything in-memory and doesn't actually connect to the database. The timer is helpful for testing network latency - in this specific case I think I was testing the loading bar.